### PR TITLE
:sparkles: new feature DatePicker

### PR DESCRIPTION
**Pull request by alexandrupopescu123**
# Description

Added Vue.js DatePicker component to form input First Brewed
+ Cursor pointer added as global style
+ Added FirstBrewedUtils -> renamed to BeerUtils with substrFirstBrewedReversedDate function
+ Resolve sort icons wrapping on resize window

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/utils/BeersUtils.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default {
+    substrFirstBrewedReversedDate: ( date ) => ( date.substring( 3, 7 )+'-'+ date.substring( 0, 2 )+ '-01' )
```
Comment (ionutt) [inline src/utils/BeersUtils.js]: rename this to getDateFromFirstBrew....... it should not matter how that is done

File: src/views/beer/Beer.vue [hunk 2]
```diff
@@ -141,6 +151,11 @@
                 return this.$route.params.id;
             },
         },
+        watch: {
+            'beerModel.first_brewed_date': function ( newValue ) {
+                this.beerModel.first_brewed = ( newValue.getUTCMonth() + 1 ) + '/' + newValue.getUTCFullYear();
```
Comment (ionutt) [inline src/views/beer/Beer.vue]: newValue.getUTCMonth() + 1 ) + '/' + newValue.getUTCFullYear()

that an be an function inside dateUtils

getDateFormatBy ( data, format ) ---- you can use moment.js. to accomplish that.

File: src/utils/BeersUtils.js (hunk 1 above)
Comment (alexandrupopescu123) [reply src/utils/BeersUtils.js]: done

File: src/views/beer/Beer.vue (hunk 2 above)
Comment (alexandrupopescu123) [reply src/views/beer/Beer.vue]: used moment.js library to replace getUTC functions in view, but couldn't use it from utils

**Pull request merged**